google-cloud-spanner/src/main/java/com/google/cloud/spanner/MetricRegistryConstants.java

```diff
@@ -84,4 +95,13 @@ class MetricRegistryConstants {
   static final String NUM_RELEASED_SESSIONS_DESCRIPTION =
       "The number of sessions released by the user and pool maintainer.";
   static final String NUM_SESSIONS_IN_POOL_DESCRIPTION = "The number of sessions in the pool.";
+
+  static final String Scope = "cloud.google.com/java/spanner";
+  static final String SPANNER_GFE_LATENCY_NAME = "gfe_latency";
+  static final String SPANNER_GFE_LATENCY_DESCRIPTION =
+      "Latency between Google's network receiving an RPC and reading back the first byte of the response";
```

Comment:
Should we consider using the word 'request' instead of 'RPC'? So like this:
```suggestion
      "Latency between Google's network receiving a request and reading back the first byte of the response";
```

Reply:
This is the existing description message for the metrics which might already be used by customers. Should we change this ?

Reply:
No, good point.